bin/install

```diff
@@ -14,13 +14,25 @@ install_maven() {
 	build_copy_cleanup $version "$destdir"
 }
 
+# Function to compare versions
+version_lt() {
+	# returns 0 (true) if first version is less than second version, otherwise 1 (false)
+	[ "$(printf '%s\n' "$1" "$2" | sort -V | head -n1)" != "$2" ]
+}
+
 # Download Maven source from Apache
 get_maven() {
 	local version=$1
 	local destdir=$2
 
 	local major=$(echo $version | cut -d '.' -f 1)
 	local base="http://dlcdn.apache.org/maven"
+
+	# Set base URL conditionally based on version
```

Comment:
... to use archive that supports older versions

Reply:
Code has been updated based off of @adamcohen2 's suggestion, so this is now outdated.